Fail to compile CAS sources on Ubuntu 20.04.1

After configuring sources, invoking `make -j` prints following warnings and due to -Werror compiler flag the whole process terminates with an error:
```
make[2]: Entering directory '/root/open-cas-linux/casadm'
  CC  cas_lib.c
In function ‘vcaslog’,
    inlined from ‘vcaslog’ at cas_lib.c:82:5:
cas_lib.c:114:12: warning: ‘strnlen’ specified bound 18446744073709551615 exceeds maximum object size 9223372036854775807 [-Wstringop-overflow=]
  114 |  timestamp[strnlen(timestamp, SIZE_MAX)-1] = 0;
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
  CC  cas_main.c
  CC  argp.c
argp.c: In function ‘run_command’:
argp.c:491:2: warning: ‘ftime’ is deprecated [-Wdeprecated-declarations]
  491 |  ftime(&t0);
      |  ^~~~~
In file included from argp.c:11:
/usr/include/x86_64-linux-gnu/sys/timeb.h:39:12: note: declared here
   39 | extern int ftime (struct timeb *__timebuf)
      |            ^~~~~
argp.c:510:2: warning: ‘ftime’ is deprecated [-Wdeprecated-declarations]
  510 |  ftime(&t1);
      |  ^~~~~
In file included from argp.c:11:
/usr/include/x86_64-linux-gnu/sys/timeb.h:39:12: note: declared here
   39 | extern int ftime (struct timeb *__timebuf)
      |            ^~~~~
```

**OS:** Ubuntu 20.04.1
**kernel:** 5.4.0-42-generic
**CAS version:** 20.09.0.0364.master (current master branch)

**Reproducible:** always

**Note:** After removing -Werror flag and compiling with warnings, CAS seems to function normally - simple fio read/write IO completes successfully.

--- a/casadm/argp.c
+++ b/casadm/argp.c
@@ -8,7 +8,7 @@
 #include "argp.h"
 #include "cas_lib.h"
 #include "cas_lib_utils.h"
-#include <sys/timeb.h>
+#include <sys/time.h>
 #include <sys/stat.h>
 
 #define PADDING "   "
@@ -472,6 +472,11 @@ void log_command(int argc, const char **argv, int result, long long int timespan
 	free(command);
 }
 
+static inline long long int timeval_to_msec(struct timeval t)
+{
+	return 1000 * t.tv_sec + t.tv_usec / 1000;
+}
+
 /**
  * run command. Additionally log its execution and report any errors if
  * they've happened
@@ -480,15 +485,12 @@ int run_command(cli_command *commands, int cmd, int argc, const char **argv)
 {
 	int result;
 	const char *syslog_path = "/var/log/messages";
-	/* time buffer and stat buffer after running command */
-	struct timeb t0;
+	struct timeval t0, t1;
 	FILE *messages_f;
-	/* time buffer and stat buffer after running command */
-	struct timeb t1;
 	long long int timespan;
 
 	/* collect time */
-	ftime(&t0);
+	gettimeofday(&t0, NULL);
 	/* collect stat buffer for syslog */
 	messages_f = fopen(syslog_path, "r");
 	if (messages_f) {
@@ -507,8 +509,8 @@ int run_command(cli_command *commands, int cmd, int argc, const char **argv)
 
 	/* execute CAS command */
 	result = commands[cmd].handle();
-	ftime(&t1);
-	timespan = (1000 * (t1.time - t0.time) + t1.millitm - t0.millitm);
+	gettimeofday(&t1, NULL);
+	timespan = timeval_to_msec(t1) - timeval_to_msec(t0);
 
 	if (commands[cmd].short_name != 'V') {
 		log_command(argc, argv, result, timespan);

--- a/casadm/cas_lib.c
+++ b/casadm/cas_lib.c
@@ -111,8 +111,6 @@ int vcaslog(int log_level, const char *template, va_list args)
 		goto out;
 	}
 
-	timestamp[strnlen(timestamp, SIZE_MAX)-1] = 0;
-
 	fseek(log, 0, SEEK_END);
 	fprintf(log, "%s casadm: ", timestamp);
 	vfprintf(log, template, args);

